bonheur/Sources/ViewControllers/TagModalView/TagSelectModalViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  TagSelectModalViewController.swift
+//  bonheur
+//
+//  Created by 김소현 on 2023/01/23.
+//
+
+import UIKit
+
+class TagSelectModalViewController: UIViewController {
+    
+    let guidingTextLabel: UILabel = {
+        let label = UILabel()
+        label.text = "해시태그 선택"
+        label.textAlignment = .center
+        label.font = UIFont(name: "SFPro-Bold", size: 16)
+        label.textColor = UIColor(red: 0.149, green: 0.15, blue: 0.149, alpha: 1)
+        return label
+    }()
+    
+    lazy var modalDismissButton: UIButton = {
+        let button = UIButton()
+        button.setBackgroundImage(UIImage(named: "ModalDismissButton"), for: .normal)
+        button.sizeThatFits(CGSize(width: 10, height: 10))
+        button.tintColor = .black
+        button.addTarget(self, action: #selector(HomeViewController.modalDismissButtonTapped), for: .touchUpInside)
+        return button
+    }()
+    
+    private var tagAddingTextField: UITextField = {
+        let textField = UITextField()
+        textField.textColor = .black
+        textField.placeholder = "해시태그 추가"
+        textField.backgroundColor = .clear
+        return textField
+    }()
+    
+    let warningTextLabel: UILabel = {
+        let label = UILabel()
+        label.text = "10자 이내로 입력 가능합니다."
+        label.font = UIFont(name: "SFPro-Regular", size: 12)
+        label.textColor = UIColor(red: 1, green: 0.341, blue: 0.196, alpha: 1)
+        return label
+    }()
+    
+    let selectCompleteButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("선택 완료", for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.backgroundColor = UIColor(red: 0.533, green: 0.846, blue: 0.07, alpha: 1)
+        button.layer.cornerRadius = 13
+        return button
+    }()
```

Comment:
P2;
selectCompleteButton에 font가 적용되지 않은 것 같아서 이렇게 수정해주시면 될 것 같습니다!

```suggestion
    let selectCompleteButton: UIButton = {
        let button = UIButton()
        button.setTitle("선택 완료", for: .normal)
        button.titleLabel?.font = UIFont(name: "SFPro-Bold", size: 16)
        button.setTitleColor(.white, for: .normal)
        button.backgroundColor = UIColor(red: 0.533, green: 0.846, blue: 0.07, alpha: 1)
        button.layer.cornerRadius = 13
        return button
    }()
```

Reply:
폰트 설정을 자꾸 빼먹네요 🥲 죄송합니다,,!
+ 와우,,, Suggested change 활용하는거 너무 좋아요! :)